app/command_utils.cc.template

```diff
@@ -13,11 +13,15 @@
 #include "brave/app/brave_command_ids.h"
 #include "brave/components/commander/common/buildflags/buildflags.h"
 #include "brave/components/commands/common/features.h"
+#include "base/no_destructor.h"
+#include "base/strings/string_util.h"
 #include "chrome/app/chrome_command_ids.h"
 #include "chrome/grit/branded_strings.h"
 #include "chrome/grit/generated_resources.h"
 #include "components/grit/brave_components_strings.h"
 #include "components/strings/grit/components_strings.h"
+#include "third_party/re2/src/re2/re2.h"
```

Comment:
Probably need to add `//third_party/re2` to deps in `app/BUILD.gn`'s `command_ids` target?

Reply:
:facepalm: